### PR TITLE
proposed CruD endpoins for Tag

### PR DESCRIPTION
**Pull request by jehrlich**
Adds POST and DELETE endpoints for Tag object and updates Tag schema to match server Tg object.

### CODE REVIEW

**Comment by jehrlich** [comment]
@VPatschka these are proposed endpoints which you are welcome to add to or edit, and merge if they will meet your needs.

**Comment by jehrlich** [comment]
@VPatschka I meant put the Diagnosis (2nd) commit on it's own branch, I guess this is OK too. You will have to modify the Admin code to remove the Diagnosis.code field.

File: src/openapi.yaml [hunk 1]
```diff
@@ -862,6 +861,55 @@ paths:
...
+              schema:
+                type: object
+                properties:
+                  protocol:
+                    $ref: "#/components/schemas/Tag"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+  /api/protocols/tags/{id}}:
```
Comment (VPatschka) [inline src/openapi.yaml]: You have small mistake here with second "}"

**Comment by VPatschka** [comment]
I think the proposed changes are fine and we can merge it after fixing the small mistake (I am not sure if I should do it, since it is your PR).

**Pull request merged**